tools/tests/README.md

```diff
@@ -19,11 +19,42 @@ run the tests in Google Chrome.
 
 # Adding new tests
 
-All tests are located in tools/tests. There is a separate folder for each component.
-The name of the folder should be the same as the name of the folder that contains the
-source of the component. For example the test folder for the radial-menu should be radial-menu.
-This is required in order to simplify the setup process that copies the source bundles into
-the test folders. All tests are named **test.js**.
+All tests are located in tools/tests. There is a separate folder for each component. All tests are named **test.js**.
+The dependencies of the test are listed in the package.json file located in tools/tests. If your tests depends on some package - add it there:
+
+tools/tests/package.json
+~~~~{.js}
+{
+    "name": "components-tests",
+    "dependencies": {
+      "coherent-gameface-automatic-grid": "^1.0.6",
+      "coherent-gameface-components": "^1.0.5",
+      "coherent-gameface-checkbox": "^1.0.5",
+      "coherent-gameface-dropdown": "^1.0.6",
+      "coherent-gameface-form-control": "^1.0.1",
+      "coherent-gameface-grid": "^1.0.2",
+      "coherent-gameface-menu": "^1.0.7",
+      "coherent-gameface-modal": "^1.0.4",
+      "coherent-gameface-progress-bar": "^1.0.4",
+      "coherent-gameface-radial-menu": "^1.0.3",
+      "coherent-gameface-radio-button": "^1.0.1",
+      "coherent-gameface-rangeslider": "^1.0.4",
+      "coherent-gameface-router": "^1.0.5",
+      "coherent-gameface-scrollable-container": "^1.0.6",
+      "coherent-gameface-slider": "^1.0.5",
+      "coherent-gameface-switch": "^1.0.1",
+      "coherent-gameface-tabs": "^1.0.6",
+      "coherent-gameface-tooltip": "^1.0.1"
+      // add your dependencies here
+    },
+    "author": "CoherentLabs",
+    "license": "ISC"
+}
+~~~~
+
+By default the tests create a symbolic links between the source of the components and the modules installed in tests/node_modules. This is necessary in order to test the local packages. If we don't create links we'll test the published npm modules. The links are not deleted between test runs. If you wish to skip the linking pass --link=false to the test script:
+
+`npm run test -- --link false`

```

Comment:
Maybe it will be better to run the tests with or without the `--link` argument and don't add a value to those argument?

`npm run test -- --link` - Will link the dependencies
`npm run test` - Will not link the dependencies

Reply:
Update to what we aggreed upon for `--no-link`.

Reply:
Yes, just the opposite - with --no-link because we want link to be default.
Changed to simply --no-link without value.